Library/Homebrew/language/python.rb

```diff
@@ -229,17 +249,29 @@ class Virtualenv
         # @param formula [Formula] the active {Formula}
         # @param venv_root [Pathname, String] the path to the root of the
         #   virtualenv
-        # @param python [String] which interpreter to use, e.g. "python" or
-        #   "python2"
+        # @param python [String, Pathname] which interpreter to use, e.g.
+        #   "python" or "python2"
+        sig { params(formula: Formula, venv_root: T.any(String, Pathname), python: T.any(String, Pathname)).void }
         def initialize(formula, venv_root, python)
           @formula = formula
           @venv_root = Pathname.new(venv_root)
           @python = python
         end
 
+        sig { returns(Pathname) }
+        def root
+          @venv_root
+        end
```

Comment:
Where is this getting used?

Reply:
I originally used it in this file before adding `site_packages`.

I was thinking it could be used in `homebrew-core` rather than hardcoding path again later on, e.g. `uhdm`: https://github.com/Homebrew/homebrew-core/blob/master/Formula/u/uhdm.rb#L49
```diff
-      "-DPython3_EXECUTABLE=#{buildpath}/venv/bin/python", *std_cmake_args
+      "-DPython3_EXECUTABLE=#{venv.root}/bin/python", *std_cmake_args
```

Though I can remove it if preferred as most cleanup is handled by using `site_packages`, e.g. `qt`
```diff
-    venv_root = buildpath/"venv"
-    venv = virtualenv_create(venv_root, python)
+    venv = virtualenv_create(buildpath/"venv", python)
     venv.pip_install resources
-    ENV.prepend_path "PYTHONPATH", venv_root/Language::Python.site_packages(python)
+    ENV.prepend_path "PYTHONPATH", venv.site_packages
```

Reply:
No, it makes sense in that case. I was only thinking about it in the context of this file but if it can simplify the code in core then I'm all for it.